spidermon/contrib/scrapy/pipelines.py

```diff
@@ -96,6 +97,21 @@ def _load_jsonschema_validator(cls, schema):
             )
         return JSONSchemaValidator(schema)
 
+    @classmethod
+    def _load_cerberus_validator(cls, schema):
+        print("WORKING at loading")
```

Comment:
Including logs are good practice, but don't use `print` for logging. Use the [logging](https://docs.python.org/3/library/logging.html) library from Python. See [here](https://github.com/vipulgupta2048/spidermon/blob/master/spidermon/contrib/actions/slack/__init__.py#L7) an example of a better way to log.

This is a minor change and you can change it later, there is nothing wrong to add development logs (with your messages), but don't forget to remove them (or include useful info that should go to production).

Reply:
I have changed it now, I think its best to change these habits of debugging early. Good to know. 
Should have sanitized the code, will do it from my next commits.